airbyte-integrations/connectors/source-declarative-manifest/README.md

```diff
@@ -0,0 +1,91 @@
+# Declarative-Manifest source connector
```

Comment:
probably worth mentioning somewhere in the readme that the connector is automatically updated and its version is expected to be synced with the CDK version. we can revisit the restriction in the future

Reply:
Updated this README significantly! [5f9f9d9](https://github.com/airbytehq/airbyte/pull/36501/commits/5f9f9d9694a5a687b0c406ff89df694e1de3612a)